src/simplejavacalculator/UI.java

```diff
@@ -59,6 +59,9 @@ public UI() {
         butCos = new JButton("Cos");
         butSin = new JButton("Sin");
         butTan = new JButton("Tan");
+        butxpowerofy=new JButton("x^y");
```

Comment:
Can you add a blank space between `=`, like `butxpowerofy = new JButton("x^y");`